src/loaders/rtm_load.c

```diff
@@ -226,8 +226,9 @@ static int rtm_load(struct module_data *m, HIO_HANDLE *f, const int start)
 			return -1;
 		}
 	}
-	
-	strncpy(mod->name, oh.name, 20);
+
```

Comment:
I wonder what the reason was for 20 limit. The loader had it that way since it was first added.

Reply:
I'm not sure what that was about either—maybe was supposed to be 0x20 or was mistaken for the tracker field length.

I just made a test RTM (albeit with Real Tracker 2.01) and it looks like 1) the name field definitely can fill the entire 32 bytes and also 2) those name fields aren't guaranteed to be null terminated.

For whatever reason GitHub won't let me attach it :/